pyproject.toml

```diff
@@ -1,3 +1,20 @@
 [build-system]
 requires = ["setuptools>=60"]
 build-backend = "setuptools.build_meta"
+
+[tool.isort]
+profile = "black"
+skip_glob = [
+    "docs/conf.py",
+    "venv*",
+    "local",
+]
```

Comment:
Does docs/conf.py still need to be skipped? If so, there should be a comment here explaining why.

isort should be run through pre-commit, for the reasons mentioned [here](https://github.com/beeware/toga/pull/1631#discussion_r1001860101). In which case, since venv and local are already gitignored, they don't need to be skipped here as well.

Reply:
I had this for historical reasons, combined with uncertainty over whether the many tox and venv folders in my project were contributing to the speed of execution. However, I believe you're right - it should be picking up the gitignore, and anecdotally, I'm not seeing any speed difference with vs without these extra lines.